ng-dev/release/publish/actions.ts

```diff
@@ -511,16 +523,33 @@ export abstract class ReleaseAction {
     });
     info(green(`  ✓   Tagged v${releaseNotes.version} release upstream.`));
 
+    let releaseBody = await releaseNotes.getGithubReleaseEntry();
+
+    // If the release body exceeds the Github body limit, we just provide
+    // a link to the changelog entry in the Github release entry.
+    if (releaseBody.length > githubReleaseBodyLimit) {
+      releaseBody =
+        `Release notes are too large to be captured here. ` +
+        `[View all changes here](${await this._getChangelogUrlForNextBranch(releaseNotes)}).`;
```

Comment:
This assumes that the changelog is always in the next branch. I don't like how this is a requirement for invoking this method (`_createGithubReleaseForVersion`), but I don't see a better way. @josephperrott any thoughts?

Reply:
Shouldn't we be able to link to the tree with the branch name, since we know the exact tag being used?

`https://github.com/${org}/${repo}/tree/${tag}/CHANGELOG.md` -> `https://github.com/angular/angular/blob/12.2.1/CHANGELOG.md`

Reply:
I was thinking about it, but was unsure because sometimes the changelog is edited afterwards. Although I'd agree that this is cleaner.